use VKV_LOGIN_COMMAND in k8s cronjobs

I'm trying to create daily cronjob to make backups

Here is the cronjob
```
apiVersion: batch/v1
kind: CronJob
metadata:
  name: vault-backup
  namespace: vault
spec:
  schedule: "0 3 * * *"
  jobTemplate:
    spec:
      template:
        spec:
          serviceAccountName: vault
          restartPolicy: OnFailure
          containers:
            - name: vkv
              image: falcosuessgott/vkv:latest
              command: ["/bin/sh", "-c"]
              args:
                - |
                  export VKV_LOGIN_COMMAND="vault write -field=token auth/kubernetes/login role=backup jwt=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
                  vkv snapshot save -d /mnt/vkv-export-$(date '+%Y%m%d%H%M%S')
              env:
                - name: VAULT_ADDR
                  value: "https://vault.example.com"
```

And error is
```
[ERROR] VAULT_TOKEN required but not set.
```

As i understood its should be ignore VALT_TOKEN if i use VKV_LOGIN_COMMAND

--- a/cmd/root_test.go
+++ b/cmd/root_test.go
@@ -66,7 +66,7 @@ func (s *VaultSuite) TestMode() {
 			expected: `e2e/ [type=kv2]
 ├── sub [v=1]
 │   └── user=********
-│
+│   
 └── sub2 [v=1]
     └── key=*****
 `,

--- a/docs/example_kubernetes.md
+++ b/docs/example_kubernetes.md
@@ -69,7 +69,7 @@ spec:
         spec:
           containers:
           - name: vkv
-            image: falcosuessgott/vkv:latest # stick to a version later
+            image: ghcr.io/falcosuessgott/vkv:latest # stick to a version later
             imagePullPolicy: IfNotPresent
             command: ["/bin/sh", "-c"]
             args:

